chemprop/cli/train.py

```diff
@@ -527,20 +540,19 @@ def normalize_inputs(train_dset, val_dset, args):
     return X_d_scaler, V_f_scaler, E_f_scaler, V_d_scaler
 
 
-def save_config(args: Namespace):
-    command_config_path = args.output_dir / "config.json"
-    with open(command_config_path, "w") as f:
-        config = deepcopy(vars(args))
-        for key in config:
-            if isinstance(config[key], Path):
-                config[key] = str(config[key])
+def save_config(parser: ArgumentParser, args: Namespace):
+    config_args = deepcopy(args)
+    for key, value in config_args.__dict__.items():
+        if isinstance(value, Path):
+            config_args.__dict__[key] = str(value)
 
-        for key in ["atom_features_path", "atom_descriptors_path", "bond_features_path"]:
-            if config[key] is not None:
-                for ind, path in config[key].items():
-                    config[key][ind] = str(path)
+    for key in ["atom_features_path", "atom_descriptors_path", "bond_features_path"]:
+        if config_args.__dict__[key] is not None:
+            for ind, path in config_args.__dict__[key].items():
+                config_args.__dict__[key][ind] = str(path)
```

Comment:
```suggestion
        if getattr(config_arg, key) is not None:
            for index, path in getattr(config_args, key).items():
                getattr(config_args, key)[index] = str(path)
```
I'm assuming `ind` is the index?